frontend/src/styling/StyledHome.ts

```diff
@@ -92,13 +101,22 @@ export const SortButton = styled.div`
     }
 `;
 
+export const ClubGrid = styled.div`
+    display: flex;
+    padding-right: 50px;
+    padding-left: 50px;
+    flex-direction: row;
+    flex-wrap: wrap;
+`;
+
 export const ClubCardContainer = styled.div`
-    float: left;
-    margin: 20px;
-    position: relative;
+    display: flex;
+    flex-direction: column;
+    margin: 10px 30px 50px 30px;
     background-color: ${colors.white};
-    width: 360px;
-    height: 200px;
+    width: 30vw;
```

Comment:
With this width, the cards look good on my laptop but become too wide on my monitor. Are we going to adjust the card size according to device sizes that you added?
Also, when the screen size changes, how would you want to photo to change? Keep the size and just center it on the card or zoom/shrink it to match the width?

Reply:
I think that we can keep the card so that it all shows but the background (that is extra in the card display area) can be filled in by black? What do you think? I can change that if we decide on it.

Reply:
Yeah sounds good!